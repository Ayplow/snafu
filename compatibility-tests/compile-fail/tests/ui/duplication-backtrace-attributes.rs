use snafu::{Backtrace, Snafu};

#[derive(Debug, Snafu)]
enum EnumError {
    AVariant {
        // Second attribute should be marked as duplicate
        #[snafu(backtrace)]
        #[snafu(backtrace)]
        my_backtrace: Backtrace,
    },
}

fn main() {}
