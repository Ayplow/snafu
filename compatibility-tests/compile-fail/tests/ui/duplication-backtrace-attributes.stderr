error: Multiple `backtrace` attributes are not supported on a field
 --> $DIR/duplication-backtrace-attributes.rs:8:17
  |
8 |         #[snafu(backtrace)]
  |                 ^^^^^^^^^
